docs/user/02-logs.md

```diff
@@ -447,7 +454,7 @@ The relevant metrics are:
 Currently, there are the following limitations for LogPipelines that are served by Fluent Bit:
 
 ### Unsupported Mode
-
+<!--- unsupported mode is not part of Help Portal docs --->
```

Comment:
Can't we consolidate all that custom features into one big section?

Reply:
I'm not sure what you mean, but we probably can. Please explain what you mean :D

Reply:
let's do that in a follow-up PR

Reply:
yep, fine. I mean to have one new heading saying "Unsupported Mode" and then list all the features available, so that you have only one big block to not sync instead of many small pieces